Added Bort to build/target/product/mainline.mk to whitelist files in /system.

diff --git a/target/product/base_vendor.mk b/target/product/base_vendor.mk
index 471340b..39326e7 100644
--- a/target/product/base_vendor.mk
+++ b/target/product/base_vendor.mk
@@ -81,3 +81,7 @@ PRODUCT_PACKAGES += \
 # /vendor. TODO(b/141648565): Don't install these unless they're needed.
 PRODUCT_PACKAGES += \
     applypatch
+
+# Memfault SDK patch version
+PRODUCT_PROPERTY_OVERRIDES += \
+    vendor.memfault.bort.version.patch=manually_patched
